packages/blade/src/components/Tooltip/TooltipContextWrapper.web.ts

```diff
@@ -0,0 +1,13 @@
+import type { CSSProperties } from 'react';
+import styled from 'styled-components';
+import { getTooltipContentWrapperStyles } from './getTooltipContentWrapperStyles';
+import type { TooltipContentWrapperProps } from './types';
+import BaseBox from '~components/Box/BaseBox';
+
+const TooltipContentWrapper = styled(BaseBox)<
```

Comment:
what does this tooltip content wrapper do?

Reply:
It's literally the main wrapper. holds the arrow, content does styling etc

<img width="211" alt="image" src="https://github.com/razorpay/blade/assets/35374649/d7ec331a-a147-409d-83a4-f8c5c4ec5a3a">